aspnetcore/fundamentals/native-aot.md

```diff
@@ -14,6 +14,7 @@ ASP.NET Core 8.0 introduces support for [.NET native ahead-of-time (AOT)](/dotne
 
 > [!WARNING]
 > In .NET 8, not all ASP.NET Core features are compatible with native AOT.
+> See [this GitHub issue](https://github.com/dotnet/core/issues/8288) for a list of know native AOT issues.
```

Comment:
Give them the list at the top of the article, not the bottom.

Reply:
know -> known